client/event.go

```diff
@@ -15,21 +15,10 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-package connection
+package client
 
-import "github.com/mysteriumnetwork/node/service_discovery/dto"
-
-type fakePromiseIssuer struct {
-	startCalled bool
-	stopCalled  bool
-}
-
-func (issuer *fakePromiseIssuer) Start(proposal dto.ServiceProposal) error {
-	issuer.startCalled = true
-	return nil
-}
-
-func (issuer *fakePromiseIssuer) Stop() error {
-	issuer.stopCalled = true
-	return nil
+// EventSubscriptionKeeper lets us subscribe to events and unsubscribe from them
+type EventSubscriptionKeeper interface {
+	Subscribe(topic string, fn interface{}) error
+	Unsubscribe(topic string, handler interface{}) error
```

Comment:
`Subscribe` has `fn` argument, but `Unsubscribe` - `handler`, what is the difference?

Reply:
And maybe we can something more clear than an empty interface?

Reply:
I've completely removed this whole interface(and file) and made the DI responsible for registering appropriate components to appropriate events.